workers/unity/Assets/Tests/EditmodeTests/Correctness/Core/CommandReceivedResponseTests.cs

```diff
@@ -0,0 +1,74 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Commands;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using Playground;
+
+namespace Improbable.Gdk.EditmodeTests.Core
+{
+    public class CommandReceivedResponseTests : MockBase
+    {
+        private const long EntityId = 101;
+        private const long LaunchedId = 102;
+
+        [Test]
+        public void ReceivedResponse_isNull_when_Response_is_not_received()
+        {
+            World.Step(world => world.Connection.CreateEntity(EntityId, GetTemplate()))
+                .Step(world => world.GetSystem<CommandSystem>().SendCommand(GetRequest()))
+                .Step((world, id) => Assert.IsFalse(world.GetSystem<CommandSystem>().GetResponse<Launcher.LaunchEntity.ReceivedResponse>(id).HasValue));
```

Comment:
does this have to be all on one line?

(same with a few other tests)

Reply:
Generally why are some things one line and other things split across lines?

Reply:
😅 I just do it in one line if its just a single function call